collections.md

```diff
@@ -116,6 +116,80 @@ Return information about the license template in use by the collection. The json
 * custom (**READ-ONLY**): can be true or false. True means that a custom license has been set for the collection otherwise the site license template is used and returned in the text attribute
 * text: contains the textual value of the license template to use for submission in the collection
 
+### Item template
+#### Retrieve Item template
+**GET /api/core/collections/<:uuid>/itemtemplate**
+
+Example: <https://dspace7.4science.it/dspace-spring-rest/#https://dspace7.4science.it/dspace-spring-rest/api/core/collections/1c11f3f1-ba1f-4f36-908a-3f1ea9a557eb/itemtemplate>
+
+It returns the item representing the item template of this collection. [See the item endpoint for more info](items.md#Single Item)
+
+#### Create Item template
+**POST /api/core/collections/<:uuid>/itemtemplate**
+
+To be used on a collection without a item template.
+The metadata is included in JSON
+
+```json
+{
+  "metadata": {
+    "dc.type": [
+      {
+        "value": "Journal Article",
+        "language": "en",
+        "authority": null,
+        "confidence": -1
+      }
+    ]
+  },
+  "inArchive": false,
+  "discoverable": false,
+  "withdrawn": false,
```

Comment:
Perhaps a silly question, but are these three fields needed in an Item Template?  I'd expect that for an Item Template `inArchive`, `discoverable` and `withdrawn` are always `false`.  What would happen if someone tried to make them `true`?

Reply:
Hi @tdonohue, they can be either false or not present.
We'll refuse setting them to true as mentioned in https://github.com/DSpace/Rest7Contract/pull/80/files#diff-40a34f27fa0507015ef983746697693eR154 and https://github.com/DSpace/Rest7Contract/pull/80/files#diff-40a34f27fa0507015ef983746697693eR163

Reply:
Thanks! I overlooked those notes in the Contract.  Marking this as resolved.